### PR TITLE
avoid potential ambiguity

### PR DESCRIPTION
**Pull request by jannes-m**
see #895

### CODE REVIEW

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -676,10 +676,13 @@ if (knitr::is_latex_output()){
 
 Imagine that we need to join the `capacity` variable in `cycle_hire_osm` onto the official 'target' data contained in `cycle_hire`.
 This is when a non-overlapping join is needed.
-The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m (note that this works with projected and unprojected data).
+The simplest method is to use the topological operator `st_is_within_distance()`, as demonstrated below using a threshold distance of 20 m.
```
Comment (Robinlovelace) [inline 04-spatial-operations.Rmd:679]: Another thought: is this more of a binary predicate than a topological operator? "Topological operator" implies to me that it's operating on, change, the topology of the geometry, whatever that means...

Comment (Robinlovelace) [reply 04-spatial-operations.Rmd:679]: Of course that comment applies equally to the old text as to your new text, just a rather late-in-the-day thought!

File: 04-spatial-operations.Rmd [hunk 2]
```diff
@@ -676,10 +676,13 @@ if (knitr::is_latex_output()){
...
+Note that this works with projected data and also with unprojected data.
+In the latter case, however, one needs to make sure that the spherical geometry engine (s2) is switched on which it is by default in **sf** (see section \@ref(s2)).
```
Comment (Robinlovelace) [inline 04-spatial-operations.Rmd:681]: :+1:

Comment (Robinlovelace) [reply 04-spatial-operations.Rmd:681]: Shorter version could be:

```
For unprojected (e.g. lon/lat CRSs such as WGS84) the spherical geometry engine (s2) must be enabled for `st_within_distance()` to work,, as it is in **sf** (see section \@ref(s2)).
```

**Review by Robinlovelace — APPROVED** [review]

File: 04-spatial-operations.Rmd (hunk 1 above)
Comment (jannes-m) [reply 04-spatial-operations.Rmd:679]: > Another thought: is this more of a binary predicate than a topological operator? "Topological operator" implies to me that it's operating on, change, the topology of the geometry, whatever that means...

Yes, let's use binary, see https://r-spatial.org/book/03-Geometries.html#sec-de9im

File: 04-spatial-operations.Rmd (hunk 2 above)
Comment (jannes-m) [reply 04-spatial-operations.Rmd:681]: Yes, way better, let's use your version. I will update as soon as I am back on my laptop, for now, I need to get some sleep. Happy new year by the way 😊

Comment (Robinlovelace) [reply 04-spatial-operations.Rmd:681]: Happy New Year :fireworks:

Comment (jannes-m) [reply 04-spatial-operations.Rmd]: I slightly rewrote: 

```
One can set the threshold distance in metric units also for unprojected data (e.g. lon/lat CRSs such as WGS84), if the spherical geometry engine (s2) is enabled, as it is in **sf** by default (see section \@ref(s2)).
```

**Pull request merged**